packages/nova-react/src/graphql/hooks.ts

```diff
@@ -193,6 +252,39 @@ export function usePaginationFragment<
   isLoadingNext: boolean;
   isLoadingPrevious: boolean;
   refetch: RefetchFn<TQuery["variables"]>;
+};
+export function usePaginationFragment<
```

Comment:
How is this definition different from one below?

Reply:
If you mean the one with the implementation then it isn't different. But afaik you can't have the implementation serve as one of the definitions.

Reply:
Aha, one is definition the other implementation